packages/editor-ui/src/components/Node.vue

```diff
@@ -207,13 +239,35 @@ export default mixins(externalHooks, nodeBase, nodeHelpers, workflowHelpers).ext
 		shiftOutputCount (): boolean {
 			return !!(this.nodeType && this.nodeType.outputs.length > 2);
 		},
-	},
+		shouldShowTooltip () : boolean {
+			return this.workflowRunning && this.isTriggerNode && this.isSingleActiveTriggerNode && !this.isTriggerNodeTooltipEmpty && !this.isNodeDisabled && !this.hasIssues && !this.isDragActive;
+		},
+ 	},
 	watch: {
 		isActive(newValue, oldValue) {
 			if (!newValue && oldValue) {
 				this.setSubtitle();
 			}
 		},
+		canvasOffsetPosition() {
+			if (this.showTriggerNodeTooltip) {
+				this.showTriggerNodeTooltip = false;
+				setTimeout(() => {
+					if (this.workflowRunning) {
+						this.showTriggerNodeTooltip = true;
+					}
```

Comment:
we should always check in the handler that we should show tooltip. otherwise we have a race condition between our two handlers.
```suggestion
					this.showTriggerNodeTooltip = this.shouldShowTooltip;
```

Reply:
Fixed